native-src/android/app/src/main/java/com/telerik/localnotifications/Builder.java

```diff
@@ -26,6 +27,10 @@
     private static final String TAG = "Builder";
     private static final String DEFAULT_CHANNEL = "Notifications";
 
+    private static final int DEFAULT_NOTIFICATION_COLOR = Color.parseColor("#ffffffff");
```

Comment:
I got these default values from [this StackOverflow answer](https://stackoverflow.com/a/34195759) which I think they got the values from the [Android source code](https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/values/config.xml#897)